libs/ledgerjs/packages/hw-app-elrond/src/Elrond.ts

```diff
@@ -111,9 +115,24 @@ export default class Elrond {
     ]);
   }
 
-  async signTransaction(path: string, message: string, usingHash: boolean): Promise<string> {
+  // kept 'usingHash' for compatibility reasons, siging by hash is the only supported way
+  async signTransaction(path: string, message: string, _usingHash?: boolean): Promise<string> {
```

Comment:
If it isn't called anymore (maybe it is, though), can be dropped.

Reply:
I am not sure whether it is or it is not called, I'd leave it for the moment.